modules/@apostrophecms/schema/ui/apos/components/AposInputSlug.vue

```diff
@@ -190,7 +190,9 @@ export default {
       const options = {
         def: ''
       };
-      if (this.field.page && !componentOnly) {
+      if (this.field.aposIsTemplate) {
+        options.allow = '@';
```

Comment:
I am not sure this is sufficient. How will users edit the template in context if it is a page template and the slug does not contain a leading `/`? I would imagine a page template slug will probably start with `/@`. We might need to modify sluggo to allow more than one exception.

Reply:
It seems we are not yet using the array support in sluggo to support both `/` and `@`. In the template module we'll want to ensure it is `/@` specifically at the beginning for pages.